internal/pkg/service/common/task/cleanup.go

```diff
@@ -14,6 +14,77 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
+const (
+	CleanupTimeout              = 5 * time.Minute
+	CleanupSuccessfulTasksAfter = 1 * time.Hour
+	CleanupFailedTasksAfter     = 48 * time.Hour
+	CleanupUnfinishedTasksAfter = 48 * time.Hour
+)
+
+// Cleanup deletes old tasks to free space in etcd.
+func (n *Node) Cleanup() (err error) {
+	logger := n.logger.AddPrefix("[cleanup]")
```

Comment:
Rovnako som presunul celu logiku cleanup do `Node`.

Rozdiel je v tom, ze uz to nie je `Task`:
- Klient aj tak nepotrebuje, tj. nemusi byt vidiet cleanup v tasks, ktore vratime napr. cez API.
- Ani sa to ^^^ vlastne neda, kedze robime cleanup vsetkych taskov naraz.
- V logoch nam to zostalo.